src/Floating.svelte

```diff
@@ -45,8 +45,12 @@
   }
 
   .toggle {
+    position: relative;
     user-select: none;
   }
+  .highlight {
+    z-index: 100;
+  }
```

Comment:
Why do we need this?

Reply:
I left this, since if you use the `overlay` option, the toggle icon ends up under the overlay, which isn't that nice..
Now that we don't use `overlay` anymore anywhere where we use the `Floating` component, we could also remove the overlay functionality all together 🤔

Reply:
Hm, maybe let's leave the overlay thing for now, I thought of using the `<Floating>` component for modals, just haven't gotten around to that refactor.